Codecademy/gamut
# fix(Markdown): filter styles from video

### Overview


Enables source tags from Video and filters out `video`s with a style prop


### PR Checklist


#### Testing Instructions

1. Head to the Markdown story
2. See the stories look correct
3. Check out the portal-app PR env
4. See the article rendering

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Portal       | [Portal Link]( https://github.com/codecademy-engineering/mono/pull/8161')  | [Portal Env](http://www.google.fr/ 'Named link title')   |
